src/Core/src/Handlers/Page/PageHandler.iOS.cs

```diff
@@ -24,24 +24,6 @@ protected override ContentView CreatePlatformView()
 			throw new InvalidOperationException($"PageViewController.View must be a {nameof(ContentView)}");
 		}
 
-		protected override void ConnectHandler(ContentView nativeView)
-		{
-			var uiTapGestureRecognizer = new UITapGestureRecognizer(a => nativeView?.EndEditing(true));
```

Comment:
Should we include with this changes an alternative way to manage the keyboard etc?